cdap-distributions/src/.dockerignore

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
Could these docker files be moved into ``src/docker/...`` ?  Or was it that the Dockerfile wouldn't work due to https://github.com/caskdata/cdap/blob/fbf3da6b103a47d9242110e6a98c59641ae314a3/cdap-distributions/src/Dockerfile#L25 or similar?